### DISCUSSION TITLE
Need help we moved rhel7 OS to rhel8 new convert not showing the title of the image as old version

### DISCUSSION THREAD

**Discussion by prsgovi** [ImageMagick/ImageMagick#7182] [General]
We had ImageMagick 6.9.10-68 Q16 x86_64 2023-09-18 version on rhel7; "convert" to a jpg file worked fine. After the rhel8 migration, we got ImageMagick 6.9.12-93 Q16 x86_64 17898 version. The new version, "convert," cropped the title a bit of the same image we tested with the old version. 

Questions 1. What is the recommended version of ImageMagick rpm for rhel8? 2. Is there any recommendations to change anything to convert JPG without cropping the heading ?

**Comment by snibgo** [comment]
Please show an example so we can understand the problem: a sample input image, and your command.

**Comment by prsgovi** [comment]
![Good_one](https://github.com/ImageMagick/ImageMagick/assets/28694751/5e845d9f-7495-4e82-a6e1-21ca8fb4e8b1)
![Bad_one](https://github.com/ImageMagick/ImageMagick/assets/28694751/5edf5cd9-a3e1-4c3b-a121-a36b42f80f22)
Please find the attached images, good_one image shows the heading which was working fine with old version on rhel7,  bad_one image cropped heading with the newer version on rhel8.

**Reply by prsgovi** [reply]
the first image is good_one, and the second is bad_one.

**Reply by dlemstra** [reply]
As Snibgo said please provide your input image and the command that you executed.

**Comment by snibgo** [comment]
@prsgovi : You have shown two outputs, but not the input, and not the command that made the outputs.

**Reply by prsgovi** [reply]
Here is the high-level steps we are following, ( working on giving the sample input files, will send in few)

1. convert PS to EPSI Output to EPSFilename
 
ps2epsi $PSFilename $EPSIFilename
 
2. Convert EPS to JPG (Rotate option with the condition 1)
               
convert -density $opts{D} -rotate '90>'  $EPSFilename $JPGFilename
 
3. Convert EPS to JPG (Rotate option with the condition p)
               
convert -density $opts{D} -rotate "90<"  $EPSFilename $JPGFilename
 
4. If no condition met ( 1 or p)
 
convert -density $opts{D} $EPSFilename $JPGFilename
 
5. Resizing JPG   

convert -resize $opts{W}x$opts{H} $JPGFilename  $JPGFilename2

**Reply by prsgovi** [reply]
the exact steps with the same input image file works fine on older version on rhel7 but not on rhel8

**Comment by snibgo** [comment]
You have shown five commands. Which of them is causing the problem? Please show the input file to that command.

**Reply by prsgovi** [reply]
here is the specific command we are having the issue

convert -density 1024 helloEPS hellopEPS.jpg

**Reply by prsgovi** [reply]
hope the above command information is good enough, let me know if you need any more details, I can provide

**Reply by dlemstra** [reply]
We have now asked you multiple times for the input image that can be used to reproduce this?
